<launch> 
    <node name="engine" pkg="adaptation_engine" type="engine" output="screen" />

    <!--<param name="frequency" value="1" /> --> <!--1 Hz the module frequency is the same as the monitors--> 
    <param name="monitor_freq" value="1" />        <!-- Hz -->

    <param name="setpoint" value="500" />          <!-- -->
    <param name="actuation_freq" value="0.02" />    <!-- Hz -->
    <param name="info_quant" value="5" />         <!-- [1,1000] -->

    <param name="offset" value="0.10" />            <!-- % of the current state -->
    <param name="gain" value="0.005" />              <!-- search granularity -->

    <param name="adaptation" value="cost" />       <!-- reliability or cost -->

</launch>